yarn.lock

```diff
@@ -2145,6 +2164,22 @@ __metadata:
   languageName: node
   linkType: hard
 
+"glob@npm:^11.0.0":
```

Comment:
I'm surprised we don't already have a glob dep...

oh... this is in agoric-3-proposals.

In agoric-sdk, we have a menagerie:

```console
$ grep 'glob ' yarn.lock 
    glob "8.0.3"
    glob "8.0.3"
    glob "8.0.3"
    glob "^8.0.1"
    glob "^7.1.6"
    is-glob "^4.0.3"
    glob "^7.1.6"
    fast-glob "^3.3.2"
    is-glob "^4.0.3"
    glob "^8.0.1"
    glob "^10.2.2"
    is-glob "~4.0.1"
    is-glob "^4.0.3"
    is-glob "^4.0.0"
    is-glob "^4.0.1"
    is-glob "^4.0.3"
    dir-glob "^3.0.1"
    fast-glob "^3.2.9"
    dir-glob "^3.0.1"
    fast-glob "^3.2.11"
    is-glob "^4.0.1"
    is-extglob "^2.1.1"
    glob "^7.1.4"
    glob "^8.0.1"
    fast-glob "3.2.7"
    glob "7.1.4"
    glob "^8.0.1"
    glob "^7.1.3"
    glob "^10.0.0"
    glob "^7.1.3"
    glob "^10.3.7"
    glob "^7.0.0"
    glob "^7.1.4"
    glob "^10.4.1"
    fast-glob "3"
```